### DISCUSSION TITLE
Alexa / ChatGPT / Home Assistant Assist Skill

### DISCUSSION THREAD

**Discussion by R4ZZB3RRY** [community/community#146709] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi!  I imported an Alexa Skill from https://github.com/alexandremendoncaalvaro/skill-alexa-chatgpt4 and I got everything to load up properly after a few trys, but after activating the skill in my Alexa app its still the same old Alexa??  Isn't the whole point of this to send and receive queries from ChatGPT?  I don't know if I'm doing anything wrong or if there is  something I overlooked, maybe someone with some experience in this could help me out a little?  

Thank you,

**Comment by mecodeatlas** [comment]
Hey @R4ZZB3RRY!

If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Comment by mecodeatlas** [comment]
Oh @R4ZZB3RRY, and you are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!
